templates/vue/cypress/support/commands.js

```diff
@@ -3,9 +3,20 @@ import roles from "./roles"
 import { deepMerge } from "./utils"
 import { API_URL, TEST_TAPESTRY_NAME } from "./constants"
 
-Cypress.Commands.add("setup", { prevSubject: false }, (fixture, role = "admin") => {
+Cypress.Commands.add("setup", { prevSubject: false }, (fixture, role = "admin", tyde = false) => {
   if (fixture) {
     cy.get(fixture).then(tapestry => {
+      // Add copilot permissions for Tyde
+      if (tyde && !fixture.startsWith("tyde/")) {
```

Comment:
I think a better way to do this would be to use a `TAPESTRY_ENV` variable inside the cypress config that we switch between the `tyde` and `master` branches accordingly:

```
{
  ...
  "env": {
    "TAPESTRY_ENV": "TYDE"
  }
  ...
}
```

You would then change this line to:

```suggestion
      if (Cypress.env("TAPESTRY_ENV") === "TYDE") {
```

And remove the `tyde` parameter at the top. This way, we don't have to define a new `tydeSetup` command, keeping the test code the same.

I also don't think we need the second check in the conditional. I think `!fixture.startsWith("tyde/")` will always be true since we're using aliases which always start with `@`.

Reply:
Thanks for the suggestion. I didn't think to use env variables. I'll go ahead and make the switch. Let me know if there are any further changes/approaches you were thinking.